### ISSUE TITLE
Ability to view and write to an existing folder

### ISSUE THREAD

**Issue by dbrandonjohnson** [Philip-Scott/Notes-up#198]
I keep all my notes in a specific folder structure that I sync across my devices with things like Seafile or NextCloud.  It would be nice to just point Notes-up to an existing folder and edit markdown files directly.  Notebooks should just be the folder structure in the target directory.

**Comment by kr428** [comment]
Yes, very much this. Notes-Up is the best note-keeping app I've seen on the Linux desktop to date, and virtually every tool for writing notes in NextCloud suck. Merging these two would be amazing.

**Comment by wolframroesler** [comment]
Really need this. I have my notes in Nextcloud, usually using Nextcloud's built-in notes app, which means that my notes are simply .md files in one particular directory. Being able to point Notes-up to that directory would be my note-taking dream come true.

**Comment by PaulB65** [comment]
+1 to adding the ability to view and write to an existing folder.
(also +1 for #274, and #322)

**Comment by BeatLink** [comment]
Any updates on this? I would literally donate on a monthly basis if this feature was supported

**Comment by gitoso** [comment]
I would also love to have this. I just like to have a git repository with all notes and it would be awesome if I could import them

**Comment by tpalucki** [comment]
+1

**Comment by sareakar** [comment]
+1

**Comment by sareakar** [comment]
+1

**Comment by davidak** [comment]
Please don't spam. Use the :+1: in the issue description.
Everyone subscribed to an issue get's an e-mail with your post and it is not constructive.
